lib/authz/permissions_test.go

```diff
@@ -917,39 +901,22 @@ func TestAuthorizeWithVerbs(t *testing.T) {
 			}),
 			errAssertion: require.NoError,
 		},
-		{
-			name: "regular auth with verbs",
```

Comment:
Did we cut this because we don't instantiate an access service anymore?

Is this covered somewhere else? (I assume so, just checking.)